### ISSUE TITLE
Hide ports from the router window

### ISSUE THREAD

**Issue by szszoke** [geoffreybennett/alsa-scarlett-gui#26]
I never use the SPDIF ports and it would be nice if I could hide it.

**Comment by geoffreybennett** [comment]
I agree. #25 is essentially the same problem. Being able to save data to the software configuration space would let us do that.
